### ISSUE TITLE
AccessibilityID missing after release build

### ISSUE THREAD

**Issue by suvro24** [microsoft/appcenter#604]
Appcenter release build APK is different from locally build APK. My App has AccessibilityID for Appium UI testing. After building Release APK using Appcenter build, my tests were failing. Then I download the apk and checked using local Appium server and there were no AccessibilityID available in the ViewTree.

My locally build APK doesn't have this issue. Can anyone help me with this issue?

Thank you.
